nixos/modules/services/mail/isync.nix

```diff
@@ -0,0 +1,72 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.isync;
+in {
+
+  options.services.isync = {
+    enable = mkEnableOption "Isync, a software to dispose your mailbox(es) as a local Maildir(s)";
+
+    install = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        Whether to install a user service for Isync. Once
+        the service is started, emails will be fetched automatically.
+
+        The service must be manually started for each user with
+        "systemctl --user start isync" or globally through
+        <varname>services.isync.enable</varname>.
+      '';
+    };
+
+    package = mkOption {
+      type = types.package;
```

Comment:
Personally I prefer only having such options when there's a reason for it, e.g. there's an alternate version or so, but people have mixed opinions on this, see https://github.com/NixOS/nixpkgs/issues/50476